upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -98,6 +100,9 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["EncodeGCELabel"] = gce.EncodeGCELabel
 
 	dest["DnsControllerImage"] = tf.DnsControllerImage
+
+	// TODO: Only for AWS?
+	dest["AWSRegion"] = tf.AWSRegion
```

Comment:
Let me add more clarity, we can only enable this for aws. You need to check the cloud provider and only set when we are running in aws.